gateway/config/standalone.lua

```diff
@@ -0,0 +1,22 @@
+local PolicyChain = require('apicast.policy_chain')
+
+local standalone = PolicyChain.load_policy(
+        'apicast.policy.standalone',
+        'builtin',
+        { url = context.configuration })
+
+if arg then -- running CLI to generate nginx config
```

Comment:
Where does this `arg` come from?

Reply:
Oh right, lua's arg.

Reply:
Yep. It is a global variable when executed through CLI.